tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -1050,6 +1050,19 @@ def find_interface(tests, id=nil, skipcheck=true)
     msg = 'Unable to find suitable interface module for this test.'
     prereq_skip(tests[:resource_name], self, msg)
   end
+
+  if type =~ /ethernet/i
```

Comment:
We have a meeting scheduled after scrum to discuss a long-term fix for testbed cleanup. I'll send you the invite.